rpc/clients/typescript/src/types.ts

```diff
@@ -396,11 +396,18 @@ export interface ValidationResults {
     rejected: RejectedOrderInfo[];
 }
 
-export interface GetOrdersResponse {
+export interface RawGetOrdersResponse {
     snapshotID: string;
+    snapshotTimestamp: string;
     ordersInfos: RawAcceptedOrderInfo[];
 }
 
+export interface GetOrdersResponse {
```

Comment:
Let's document this new exported interface, especially the fact that `snapshotTimestamp` is seconds since the Unix Epoch in UTC.